### PR TITLE
template: oci template supports for char user info

### PR DESCRIPTION
**Pull request by ssup2**
oci template changes character user info to uid, gid
according to OCI image spec.

Signed-off-by: Jungsub Shin jungsub_shin@tmax.co.kr

### CODE REVIEW

**Comment by brauner** [comment]
//cc @flx42, @3XX0 :)

**Comment by brauner** [comment]
jenkins: test this please

**Comment by ssup2** [comment]
Is there any feedback, or review??

**Comment by ssup2** [comment]
Ok. If there is no feedback, What should i do to verify this commit??

File: templates/lxc-oci.in [hunk 1]
```diff
@@ -129,23 +129,40 @@ getenv() {
...
+  if ! isdecimal "${user}" && [ -f ${passwdpath} ]; then
+    user=$(grep "^${user}:" "${passwdpath}" | awk -F: '{print $3}')
+  fi
+
+  group=${usergroup[1]:-}
+  if [ -z "${group}" ] && [ -f "${passwdpath}" ]; then
+    group=$(grep "^[^:]*:[^:]*:${user}:" "${passwdpath}" | awk -F: '{print $4}')
+  elif ! isdecimal "${group}" && [ -f "${grouppath}" ]; then
+    group=$(grep "^${group}:" "${grouppath}" | awk -F: '{print $3}')
+  fi
```
Comment (stgraber) [inline templates/lxc-oci.in]: So the problem with this approach is that it assumes a system where /etc/passwd and /etc/group is used, this isn't guaranteed and depends on NSS configuration.

Any reason why you're not using `getent passwd` and `getent group` instead?

**Pull request merged**